### PR TITLE
feat: add package request template

### PR DESCRIPTION
**Pull request by castrojo**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    validations:
+      required: true
+  - type: dropdown
+    id: image
+    attributes:
+      label: Image
+      description: Which specific image do you want?
+      options:
+        - main
+        - nvidia
```
Comment (marcoceppi) [inline .github/ISSUE_TEMPLATE/feature_request.md:26]: Isn't nvidia going to be in a different repo? Wouldn't this instead be which specific distro? `silverblue`, `kinoite`, `all`, etc

**Review by marcoceppi — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (castrojo) [reply .github/ISSUE_TEMPLATE/feature_request.md:26]: I was unsure what to put there so thanks, I'll fix it. 🗡️

Comment (castrojo) [reply .github/ISSUE_TEMPLATE/feature_request.md]: PTAL

**Review by bigpod98 — APPROVED** [review]

**Review by joshua-stone — APPROVED** [review]
LGTM

**Review by marcoceppi — APPROVED** [review]

**Pull request merged**